Remotion/Web/ClientScript/Scripts/TreeView.ts

```diff
@@ -103,14 +103,15 @@ class WebTreeView
 
   private static OnKeyDown (event: KeyboardEvent, treeView: HTMLElement): void
   {
-    const treeNodes = Array.from (treeView.querySelectorAll<HTMLElement> ('li[role=treeitem]'));
+    const treeNodes = Array.from (treeView.querySelectorAll<HTMLElement> ('a[role=treeitem]'));
+    const parentTreeNodeListItems = Array.from(treeNodes.map<HTMLElement>(a => a.closest('li')!));
```

Comment:
naming: this sounds like it deals with the parent-TreeNode, not with the containing-LI of this treenode.
-> `treeNodeParentListItems`